Cargo.toml

```diff
@@ -12,7 +12,7 @@ exclude     = [
 ]
 
 [dependencies]
-libc = { git = "https://github.com/rust-lang/libc" }
+libc = { git = "https://github.com/rust-lang/libc", branch = "master" }
```

Comment:
The `branch` specification isn't necessary.  Please revert changes to this file.